pub_sub/csharp/order-processor/Program.cs

```diff
@@ -0,0 +1,36 @@
+using System.Text.Json.Serialization;
+
+var builder = WebApplication.CreateBuilder(args);
+builder.Services.AddEndpointsApiExplorer();
+builder.Services.AddSwaggerGen();
+
+var app = builder.Build();
+
+if (app.Environment.IsDevelopment())
+{
+    app.UseSwagger();
+    app.UseSwaggerUI();
+    app.UseDeveloperExceptionPage();
+}
+
+app.MapGet("/dapr/subscribe", () => {
+    var sub = new DaprSubscription(pubsub_name: "order_pub_sub", topic: "orders", route: "orders");
+    Console.WriteLine("Dapr pub/sub is subscribed to: " + sub);
+    return Results.Json(new DaprSubscription[]{sub});
+});
+
+app.MapPost("/orders", (DaprData<Order> request_data) => {
+    Console.WriteLine("Subscriber received : " + request_data.data.order_id);
+    return Results.Ok(request_data.data.order_id);
+});
+
+await app.RunAsync();
+
+app.Run();
+
+public record DaprData<T> ([property: JsonPropertyName("data")] T data); 
+public record Order([property: JsonPropertyName("orderid")] int order_id);

```

Comment:
please do a pass on naming of properties/parameters/variables/json names . We want the sample to feel native to a .NET developer.

Reply:
Done